src/components/Header/Header.module.css

```diff
@@ -0,0 +1,17 @@
+.loadFactoryHeader {
+  padding-bottom: 0;
```

Comment:
You should use `PageSection` properties to add/remove paddings and margins. See https://www.patternfly.org/v4/components/page/react/#main-section-padding

Reply:
It is needed for showing a real message instead of ```...[object][object].```
I need to remove only one padding to be consistent with the workspace details header.
It is better than remove all and set custom three.

If you have a better solution just add here.

Reply:
removed